packages/TradeIn/package.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "@im-ui/trade-in",
+  "version": "1.0.0",
+  "types": "dist/cjs/index.d.ts",
+  "module": "dist/esm/index.js",
+  "main": "dist/cjs/index.js",
+  "author": "InstaMotion Retail GmbH",
+  "maintainers": [
+    "Karlo S"
+  ],
+  "repository": "git@github.com:Instamotion/im-components.git",
+  "license": "MIT",
+  "scripts": {
+    "build": "npm run clean && npm run compile:esm && npm run compile:cjs",
+    "clean": "rimraf ./dist",
+    "compile:esm": "tsc -p build/tsconfig.esm.json",
+    "compile:cjs": "tsc -p build/tsconfig.cjs.json",
+    "prepublishOnly": "bolt build"
+  },
+  "files": [
+    "dist"
+  ],
+  "dependencies": {
+    "@im-ui/theme": "^2.0.8",
+    "@im-ui/utils": "^0.5.5",
+    "@im-ui/styled-dropdown": "^1.3.0"
```

```suggestion
    "@im-ui/theme": "^2.0.10",
    "@im-ui/utils": "^0.5.6",
    "@im-ui/styled-dropdown": "^1.3.0"
```
We should use the up to date versions